examples/ProjectExample/myProject.py

```diff
@@ -1,6 +1,6 @@
-from lib.install_packages import install
+from pypackageinstaller import install
 install()
 
 from PIL import Image
-img = Image.open("testImage.png")
+img = Image.open("examples/ProjectExample/testImage.png")
```

Comment:
Pretty sure this should be a problem. This would only happen if you run from the top folder of the repo. If running from this folder it should be fine.

Reply:
That's really weird, I'll take your word for it though, I might be running the program wrong.
![Screen Shot 2020-04-28 at 8 50 39 AM](https://user-images.githubusercontent.com/55749227/80489119-5f158180-892d-11ea-9487-32b32132f866.png)

Reply:
If you run it the way you did from vscode, you get the error, it's because the command is running relative to the project instead of the file. Changing the folder and running from there works.

![image](https://user-images.githubusercontent.com/33532035/80527929-242e4080-8963-11ea-844a-115a7188e9c4.png)